lib/LLVMIRCodeGen/LLVMIRGen.cpp

```diff
@@ -2006,6 +2006,41 @@ void LLVMIRGen::generateLLVMIRForInstr(llvm::IRBuilder<> &builder,
     break;
   }
 
+  case Kinded::Kind::BatchedReduceMaxInstKind: {
```

Comment:
This could also be trivially merged with the above case for `BatchedReduceMinInst`, right?